src/Service/MilvusVectorStoreService.php

```
@@ -271,6 +271,35 @@ public function insertProductChunks(Product $product, array $chunks): bool
+    /**
+     * Delete all vectors that belong to the given product ID.
+     */
+    public function deleteProductVectors(int $productId): bool
+    {
+        $this->logger->info('Deleting existing vectors for product', [
+            'collection_name' => $this->collectionName,
+            'product_id' => $productId,
+        ]);
+
+        try {
+            $this->milvus->vector()->delete(
```

PHPStan (2.1.15)

Missing parameter $id (array|int|string) in call to method HelgeSverre\Milvus\Resource\VectorOperations::delete().


(argument.missing)